src/isPalindrome.js

```diff
@@ -0,0 +1,5 @@
+export default function isPalindrome(string){
+  const s1 = string.replace(/\s/g,'').replace(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g,"").toLowerCase();
+  const s2 = s1.split('').reverse().join('');
+  return s2 === s1
+}
```

Comment:
This is brilliant, excellent use of regexp, and both string and array functionalities, as to return the result of the comparison between the two strings